doc/_layouts/default.html

```diff
@@ -7,6 +7,8 @@
     <!-- TODO: Restrict font variants to what we are actually using! -->
     <!-- link href='https://fonts.googleapis.com/css?family=Arimo:700italic,700,400italic,400|Cousine:700italic,700,400italic,400' rel='stylesheet' type='text/css'-->
 
+    <link rel="icon" type="image/x-icon" href="https://avatars.githubusercontent.com/u/1610185?s=400&v=4">
+    <link rel="apple-touch-icon" type="image/x-icon" href="https://avatars.githubusercontent.com/u/1610185?s=180&v=4">
```

Comment:
The media `type` is wrong, the `sizes` attribute is missing, I think we want at least 16x16, 32x32, 48x48 (svg would be even better, but requires more work).

I'm gonna close this favor of https://github.com/hspec/hspec/pull/904.